framework/wazuh/core/cluster/master.py

```diff
@@ -697,7 +698,7 @@ async def sync_integrity(self, task_id: str, received_file: asyncio.Event):
                                                                            files_metadata, self.name)
 
         total_time = (datetime.utcnow() - date_start_master).total_seconds()
-        self.extra_valid_requested = bool(worker_files_ko['extra_valid'])
+        self.extra_valid_requested = bool(worker_files_ko['TYPE'] if 'TYPE' in worker_files_ko else False)
```

Comment:
We should avoid increasing the complexity of the code, it is not appropriate that to deactivate a functionality we have to apply this type of changes to others.